src/components/WearablePreview/WearablePreview.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import './WearablePreview.css'
+
+export type WearablePreviewProps = {
+  contractAddress: string
+  tokenId?: string
+  itemId?: string
+}
+
+export class WearablePreview extends React.PureComponent<WearablePreviewProps> {
+  static baseUrl = 'https://wearable-preview.decentraland.org'
```

Comment:
As the URL is fixed, and the peer URL is also fixed in the [wearable-preview repository](https://github.com/decentraland/wearable-preview/blob/f81560b1aab6039ec139e7ac509165420d65c9d9/src/lib/api/peer.ts#L3) how are we going to render items in the dev environment?